<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PlanningProfilactic</class>
 <widget class="QDialog" name="PlanningProfilactic">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>368</width>
    <height>154</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Планирование профилактических мероприятий</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="6" column="5">
    <spacer name="horizontalSpacer_2">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="6" column="4">
    <widget class="QPushButton" name="btnClose">
     <property name="text">
      <string>Закрыть</string>
     </property>
    </widget>
   </item>
   <item row="6" column="0" colspan="3">
    <widget class="QPushButton" name="btnStart">
     <property name="text">
      <string>Начать</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="lblOrgStructure">
     <property name="text">
      <string>год</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0" colspan="5">
    <widget class="QCheckBox" name="chkAddDispExempts">
     <property name="text">
      <string>Включать диспан. фед. льготников</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1" colspan="2">
    <widget class="QSpinBox" name="sbYear">
     <property name="minimum">
      <number>2017</number>
     </property>
     <property name="maximum">
      <number>2099</number>
     </property>
    </widget>
   </item>
   <item row="1" column="0" colspan="5">
    <widget class="QCheckBox" name="chkAddDisp">
     <property name="text">
      <string>Включать диспан. по возрастам</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="4" column="0" colspan="5">
    <widget class="QCheckBox" name="chkAddProf">
     <property name="text">
      <string>Включать профилактич. осмотры</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="5" column="0">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
